src/Microsoft.Data.Sqlite.Core/Microsoft.Data.Sqlite.Core.csproj

```diff
@@ -17,7 +17,7 @@ Microsoft.Data.Sqlite.SqliteException
 Microsoft.Data.Sqlite.SqliteFactory
 Microsoft.Data.Sqlite.SqliteParameter
 Microsoft.Data.Sqlite.SqliteTransaction</Description>
-    <TargetFrameworks>net451;netstandard1.2</TargetFrameworks>
+    <TargetFrameworks>net46;netstandard1.2</TargetFrameworks>
```

Comment:
The lowest planned .NET Framework target is currently .NET 4.6.1. I think if we have strong evidence to suggest it's worth targeting anything lower than that, let's bring it up with @DamianEdwards .